packages/react-instantsearch-core/package.json

```diff
@@ -33,12 +33,13 @@
     "build": "yarn build:cjs && yarn build:es && yarn build:umd",
     "build:cjs": "babel src --root-mode upward --out-dir dist/cjs --ignore '**/__tests__/**/*.js','**/__mocks__/**/*.js' --quiet",
     "build:es": "BABEL_ENV=es babel src --root-mode upward --out-dir dist/es --ignore '**/__tests__/**/*.js','**/__mocks__/**/*.js' --quiet",
-    "build:umd": "BABEL_ENV=es rollup -c rollup.config.js",
+    "build:umd": "BABEL_ENV=rollup rollup -c rollup.config.js",
     "test": "jest",
     "release": "yarn clean && yarn build && yarn publish --non-interactive",
     "release:beta": "yarn clean && yarn build && yarn publish --tag beta --non-interactive"
   },
   "dependencies": {
+    "@babel/runtime": "^7.1.2",
```

Comment:
is there some way a user might end up with multiple babel runtimes because it's a dependency?

Reply:
Yes it's a possibility like any other dependencies. I've used an older version to minimize this. But like with any other dependencies it might happen. The behaviour that we had previously is the same as having multiple `@babel/runtime` since the helpers are inlined inside the source code.